client/src/components/Workflow/TrsImport.vue

```diff
@@ -23,6 +23,9 @@
                 <trs-tool :trs-tool="trsTool" v-if="trsTool" @onImport="importVersion(trsTool.id, $event)" />
             </div>
         </b-card>
+        <b-alert v-else show variant="danger" class="text-center my-2"
+            >Only registered users can import workflows!</b-alert
```

Comment:
Can you rephrase this to `Anonymous user cannot import workflows, please log in.` or something similar ? That seems like it would be easier to understand for users.